packages/nc-gui/utils/groupbyUtils.ts

```
@@ -0,0 +1,119 @@
+    UITypes.CreatedTime,
+    UITypes.LastModifiedTime,
+    UITypes.CreatedBy,
+    UITypes.LongText,
+    UITypes.LastModifiedBy,
+  ].includes(<UITypes>(typeof colOrUidt === 'object' ? colOrUidt?.uidt : colOrUidt))
+}
+
+// a method to parse group key if grouped column type is LTAR or Lookup
+// in these 2 scenario it will return json array or `___` separated value
+export const parseKey = (group: Group | CanvasGroup) => {
+  let key = (group?.key ?? group?.value).toString()
```

Biome (1.9.4)

[error] Unsafe usage of optional chaining.

If it short-circuits with 'undefined' the evaluation will throw TypeError here:


(lint/correctness/noUnsafeOptionalChaining)